src/pages/BirthDay.jsx

```diff
@@ -12,16 +14,14 @@ const BirthDay = () => {
           <Typography fontSize="20px">생일!</Typography>
         </TitleWrapper>
         <MonthChip>
```

Comment:
```suggestion
        <MonthChipWrapper>
```
여긴 chip들을 감싸는 부분이니 wrapper나 container같은 컴포넌트 네이밍이 적합